mage_ai/data_cleaner/transformer_actions/dependency_resolution.py

```diff
@@ -10,11 +10,19 @@
 """
 
 
-def default_resolution(df: DataFrame) -> Tuple[bool, str]:
+def default_resolution(df: DataFrame, action: Dict) -> Tuple[bool, str]:
     return True, None
 
 
-def resolve_filter_action(df: DataFrame) -> Tuple[bool, str]:
+def isolate(action_code):
```

Comment:
where is it used?

Reply:
It's currently not used, I forgot to remove, will remove now.